src/comparator/wrappers.py

```diff
@@ -42,9 +49,7 @@ def __init__(self, value, source_code_locations, path):
 
     @property
     def source_code_line(self):
-        if self.path not in self.source_code_locations:
-            return f"No source code line can be identified by path {self.path}."
-        return self.source_code_locations[self.path].span[0] + 1
+        _get_source_code_line(self.source_code_locations, self.path)
```

Comment:
Don't these need a `return` statement?
(but I'm fairly ignorant about python, so I don't really know!)

Reply:
You are right! And that helped me fix the test failures. Thanks for catching!